### ISSUE TITLE
Accessibility scan results include unexpected results

### ISSUE THREAD

**Issue by adarshrema** [microsoft/axe-pipelines-samples#154]
## Recommended changes:
Update index.test.ts (Typescript sample), SamplePageTests.cs (C# sample) files to make .withtags more prominent by including,
.WithTags("wcag2a", "wcag2aa", "wcag21aa")
and accompanying instructions in the (respective) first test case:
* typescript sample: // This test case shows the most basic example: run a scan, fail the test if there are any failures.
* C# sample: // This test case shows the most basic example: run an accessibility scan on a page and assert that there are no violations.
